contracts/BSCValidatorSet.sol

```diff
@@ -744,7 +749,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     // 2. get unjailed validators from validatorSet
     unjailedValidatorSet = new Validator[](_validatorSet.length - numOfFelony);
     i = 0;
-    for (uint index = 0; index < _validatorSet.length; index++) {
+    for (uint index = 0; index < _validatorSet.length; ++index) {
```

Comment:
`for (uint index; index < _validatorSet.length; ++index) {` ?

Reply:
fixed